src/test/scala/org/broadinstitute/dsde/agora/server/AgoraTestSuite.scala

```diff
@@ -28,7 +28,9 @@ class AgoraTestSuite extends Suites(
   new PermissionIntegrationSpec,
   new MethodDefinitionIntegrationSpec,
   new AssociatedConfigurationIntegrationSpec,
+  new CompatibleConfigurationIntegrationSpec,
```

Comment:
this has nothing to do with the current PR, but, uh, we were not running those tests at all.